### PR TITLE
[Console] Dispatch `ConsoleTerminateEvent` when exiting on signal

### LINKED ISSUE #19016: [Console] Dispatch `ConsoleTerminateEvent` when exiting on signal
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#52038
| PR author(s) | @HeahDude
| Merged in    | 6.4

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19016

### CODE REVIEW

File: components/console/events.rst [hunk 1]
```diff
@@ -152,6 +152,17 @@ Listeners receive a
     It is then dispatched just after the ``ConsoleEvents::ERROR`` event.
     The exit code received in this case is the exception code.
 
+    Additionally, the event is dispatched when the command is being exited on
+    a signal. You can learn more about signals in the
+    :ref:`the dedicated section <console-events_signal>`.
+
+    .. versionadded:: 6.4
+
+        The dispatching of the ``ConsoleEvents::TERMINATE`` event on exit
```
Comment (OskarStark) [inline components/console/events.rst]:
```suggestion
        Dispatching the ``ConsoleEvents::TERMINATE`` event on exit
```

**Review by OskarStark — APPROVED** [review]
After my comment

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Comment by alexandre-daubois** [comment]
Thanks for the fix!

**Pull request merged**